### PR TITLE
[chore]: reorganize Tests crate

### PR DESCRIPTION
**Pull request by TAdev0**
This PR reorganizes Tests crate because all files inside it except tests.cairo where not compiled when running `scarb test`.

Now everything compiles, but the scripts to generate will still need improvements

### CODE REVIEW

File: tests/utils.cairo [hunk 1]
```diff
@@ -10,13 +10,14 @@ fn hex_to_byte(h: u8) -> u8 {
     0
 }
 
-pub fn from_base16(hexs: @ByteArray) -> ByteArray {
+pub fn from_base16(hexs: @ByteArray) -> @ByteArray {
```
[the remark this answers is not in the document]
Comment (TAdev0) [reply tests/utils.cairo]: @maciejka  done

**Review by maciejka — CHANGES_REQUESTED** [review]
Please add get_blocks.sh script: https://github.com/keep-starknet-strange/raito/pull/52#discussion_r1713471255

**Comment by TAdev0** [comment]
@maciejka  its all good.

I tested it locally, it returns blocks with valid syntax/organization that compile without modification.

Is just missing the comment which will be deleted each time we regenerate the blocks. Might need a bit of help on that, dont know how to do that..

**Comment by TAdev0** [comment]
```
tristan@Tristans-MacBook-Pro raito % ./scripts/data/get_blocks.sh


Getting block: 000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f
Getting block: 00000000d1145790a8694403d4063f323d499e655c83426834d4ce2f8dd4a2ee
All blocks retrieved successfully.
```

**Comment by maciejka** [comment]
> @maciejka its all good.
> 
> I tested it locally, it returns blocks with valid syntax/organization that compile without modification.
> 
> Is just missing the comment which will be deleted each time we regenerate the blocks. Might need a bit of help on that, dont know how to do that..

Just add comments to the get_blocks.sh for now.

File: scripts/data/get_blocks.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+get_single_block() {
+    local block_hash=$1
+    ./scripts/data/get_block.sh "$block_hash"
+}
+
+main() {
+    local block_hashes=(
+        "000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"  # Genesis block
+        "00000000d1145790a8694403d4063f323d499e655c83426834d4ce2f8dd4a2ee"  # Block 170
```
Comment (maciejka) [inline scripts/data/get_blocks.sh]: Are you going to add more blocks?

**Comment by TAdev0** [comment]
@maciejka  done

File: scripts/data/get_blocks.sh (hunk 2 above)
Comment (TAdev0) [reply scripts/data/get_blocks.sh]: i was thinking of opening a dedicated PR for this

Comment (TAdev0) [reply scripts/data/get_blocks.sh]: as your prefer!

Comment (maciejka) [reply scripts/data/get_blocks.sh]: I prefer another pr.

**Review by maciejka — APPROVED** [review]

**Pull request merged**